addon/components/ui-calendar/template.hbs

```diff
@@ -1,54 +1,55 @@
-<nav local-class="nav">
-  <button 
-    type="button" 
-    local-class="nav-control on-previous-month" 
+<nav local-class="nav" class="ui-calendar-nav">
+  <button
+    type="button"
+    local-class="nav-control on-previous-month" class="ui-calendar-nav-previous"
     {{on "click" (fn this.onMoveCenter -1 "month")}}
   >
-    {{ui-icon "arrow-left-12" size="small"}}
+    {{svg-jar "arrow-left-1" height="15" width="15"}}
   </button>
-  <div local-class="nav-title">
+  <div local-class="nav-title" class="ui-calendar-nav-title">
     {{moment-format this.center "MMMM YYYY"}}
   </div>
-  <button 
+  <button
     type="button"
-    local-class="nav-control on-next-month" 
+    local-class="nav-control on-next-month" class="ui-calendar-nav-next"
     {{on "click" (fn this.onMoveCenter 1 "month")}}
   >
-    {{ui-icon "arrow-right-12" size="small"}}
+    {{svg-jar "arrow-right-1" height="15" width="15"}}
   </button>
 </nav>
 
-<div local-class="weekdays">
-  <div local-class="weekday">
+<div local-class="weekdays" class="ui-calendar-weekdays">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Sun
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Mon
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Tue
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Wed
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Thu
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Fri
   </div>
-  <div local-class="weekday">
+  <div local-class="weekday" class="ui-calendar-weekday">
     Sat
   </div>
 </div>
 
-<div local-class="weeks">
+<div local-class="weeks"  class="ui-calendar-weeks">
   {{#each this.weeks as |week|}}
-    <div local-class="week">
+    <div local-class="week" class="ui-calendar-week">
       {{#each week.days as |day|}}
-        <button 
-          type="button" 
+        <button
+          type="button"
           local-class="day {{if day.isSelected "is-selected"}} {{if day.isPreviousMonth  "is-previous-month"}} {{if day.isNextMonth  "is-next-month"}} {{if day.isToday "is-today"}} {{if day.isRangeStart "is-range-start"}} {{if day.isRangeFinish "is-range-finish"}}"
+          class="ui-calendar-day {{if day.isSelected "is-selected"}} {{if day.isPreviousMonth  "is-previous-month"}} {{if day.isNextMonth  "is-next-month"}} {{if day.isToday "is-today"}} {{if day.isRangeStart "is-range-start"}} {{if day.isRangeFinish "is-range-finish"}}"
```

Comment:
Do we need to expose all of these classes? Can we just start with `ui-calendar-day`?

Reply:
My thinking was that this would make it much easier to add additional styles in the app using ember-assembly

Reply:
Yeah I hear you, it just seems like we are repeating ourselves here - theres going to be a lot of classes on each day. Do we need to refactor UiCalendar styles to use the global classes here instead of the local ones in that case?

Reply:
Yeah that might be the best way to do it, I shall make that change!

Reply:
Hold up, sorry, I didnt know you were doing this. I was more discussing it at this point.

Reply:
I think its probably best to have it as it was before. If we do it this way, then we are kind of invalidating the point of using local class anywhere in ember-assembly.

I think best to have it with the repetition to expose the utility classes.